src/main/resources/db/migration/v0.13.0.1__agr_curation_api.sql

```diff
@@ -19,6 +19,10 @@ ALTER TABLE diseaseannotation
 	ALTER COLUMN uniqueid
  		SET NOT NULL;
 	
+ALTER TABLE diseaseannotation
+	ALTER COLUMN uniqueid
+		SET NOT NULL;
+	
```

Comment:
If this is a change to an existing file. It will not get rerun against alpha... so it would need to go in a new file.

Reply:
Duplication caused by merging of local branches - now removed